src/video/allegro_v.cpp

```diff
@@ -535,6 +535,7 @@ void VideoDriver_Allegro::MainLoop()
 			DrawMouseCursor();
 			DrawSurfaceToScreen();
 		}
+		DrawWindows();
```

Comment:
I find it now a bit odd that we do `DrawWindows` after `DrawSurfaceToScreen`. It is also called in both flows, so possibly there is some profit to be gained here